### ISSUE TITLE
RUF006 not emitted for low-level `loop.create_task`

### ISSUE THREAD

**Issue by layday** [astral-sh/ruff#3237]
AFAIK, the same pitfall applies to the lower level `loop.create_task`:

```py
import asyncio


async def foo():
    asyncio.create_task(bar())  # Warning here
    loop = asyncio.get_running_loop()
    loop.create_task(bar())  # No warning here

async def bar():
    pass

```

It would be great if ruff could support this too in some way.

Ruff version: 0.0.252

**Comment by charliermarsh** [comment]
This requires more static analysis than we're currently capable of doing, but I'd like to support these kinds of checks so I'm leaving this open.
